suse_openstack_cloud

```diff
@@ -250,6 +250,16 @@ find_and_pconf_files /srv/tftpboot/discovery/pxelinux.cfg -type f
 find_and_pconf_files /srv/tftpboot/nodes                  -type f
 
 #############################################################
+section_header "Crowbar database"
+
+crowbar_db_dump_dir=/var/log/crowbar/db-dump
+# dump the database tables
+sqlite3 /opt/dell/crowbar_framework/db/production.sqlite3 .dump > $crowbar_db_dump_dir/production.dump.sql
```

Comment:
I think https://github.com/yamldb/yaml_db is what you're looking for.  I agree it would be nicer to go through ActiveRecord, however if it introduces significant overhead then I think it would be find to define our own Rake task which calls sqlite3.  Either way it's cleaner than hardcoding implementation details in this plugin.